packages/containers/src/ActionFile/ActionFile.component.js

```diff
@@ -0,0 +1,41 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { ActionFile as PureActionFile } from '@talend/react-components';
+import actions from '../actionAPI';
+
+/**
+ * @param {Object} props react props
+ * @example
+<Action name="menu:demo"></Action>
+ */
+function Action({ name, ...rest }, context) {
+	let action;
+
+	if (name) {
+		action = Object.assign(
+			{},
+			rest,
+			actions.getProps(context, name, rest.model),
+		);
+	} else {
+		action = actions.getProps(context, rest, rest.model);
+	}
```

Comment:
Can't you remove one of the cases by assigning `actions.getProps(context, rest, rest.model)` right when you declare it?